EXILED/Exiled.API/Features/Items/Firearm.cs

```diff
@@ -55,6 +52,11 @@ public Firearm(BaseFirearm itemBase)
             : base(itemBase)
         {
             Base = itemBase;
+            foreach (ModuleBase mod in itemBase.Modules)
+            {
+                if (mod is IPrimaryAmmoContainerModule primary)
+                    ammoModule = primary;
+            }
```

Comment:
Since nothing else is happening within this foreach loop, you can remove the bracing.

```suggestion
            foreach (ModuleBase mod in itemBase.Modules)
                if (mod is IPrimaryAmmoContainerModule primary)
                    ammoModule = primary;
```

Reply:
The foreach is still a multi-line statement.
Multi-line statements, even if it's only performing one thing, should still be wrapped in braces.

https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1519.md

Reply:
i prefer it being wrapped anyways